app/routes.py

```diff
@@ -20,27 +20,52 @@
     template_folder='../web/dist',
 )
 
+cookie = {}
+
 @bp_web_api.route('/ping', methods=['GET'])
 def ping():
     return jsonify('Pong!')
 
-@bp_web_api.route('/data', methods=['GET'])
-def get_data():
+@bp_web_api.route('/auth/login', methods=['GET', 'POST'])  
+def login():   
+    if request.method == 'GET':  
+           return '''
+     <form action='login' method='POST'>
+     <input type='text' name='name' id='name' placeholder='name'/>
+     <input type='password' name='credential' id='credential' placeholder='credential'/>
+     <input type='submit' name='submit'/>
+     </form>
```

Comment:
## Warn

### Description

Redundant and undefined in Web API Spec

### Solution

```suggestion
```